src/common/components/hosts/EditHostForm.tsx

```diff
@@ -25,17 +26,15 @@ import { canHostnameBeChanged } from './utils';
 import GridGap from '../ui/GridGap';
 import { EditHostFormValues } from './types';
 import { useTranslation } from '../../hooks/use-translation-wrapper';
-import { TFunction } from 'i18next';
 
 export type EditHostFormProps = {
   host: Host;
   inventory: Inventory;
   usedHostnames: string[] | undefined;
   onCancel: () => void;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  onSave: (values: EditHostFormValues) => Promise<any>;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  onFormSaveError?: (e: any) => void | string;
+  onSave: (values: EditHostFormValues) => Promise<unknown>;
+  onHostSaveError?: (e: Error) => void;
+  getEditErrorMessage?: (e: Error) => string;
```

Comment:
The function that was previously called `onFormSaveError` was returning the title of the error, so I renamed it because I wanted to have a "real" `onFormSaveError`

Reply:
It's being rework, so: ok